pcap.pyx

```diff
@@ -52,9 +52,14 @@ cdef extern from "pcap.h":
 ctypedef void (*pcap_handler)(void *arg, pcap_pkthdr *hdr, char *pkt)
 
 cdef extern from "pcap.h":
-    pcap_t *pcap_open_live(char *device, int snaplen, int promisc,
```

Comment:
Since this is a function that is exposed by the API and users will probably expect it we should leave it.